mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -502,12 +502,12 @@ func resourceMongoDBAtlasClusterCreate(d *schema.ResourceData, meta interface{})
 	}
 
 	var containerId string
-	if providerName == "AZURE"{
+	if providerName == "AZURE" {
 		if v, ok := d.GetOk("container_id"); ok {
 			containerId = v.(string)
 			_, res, err := conn.Containers.Get(context.Background(), projectID, containerId)
```

Comment:
This code doesn't work, containerId is not set because the attribute is computed, you need to validate if a container exists if not, you need to wait until it creates automatically in order to get the containerId and set it to the `container_id` computed attribute.

Reply:
I see, I misunderstood the behavior then, let me change it, thank you paco